lab2 upload

galexandrus opened the pull request:
It includes 4 tasks

galexandrus committed:
lab2 upload

It includes 4 tasks

EgorOb commented on topic2_lab/main_task4.py:
```diff
@@ -0,0 +1,13 @@
+chars = 25  # TODO ввести количество символов в строке
+
+total_chars = chars * lines * pages  # TODO общее количество символов в книге
+total_bytes = total_chars * 1  # TODO размер одной книги в байтах
```
`total_bytes = total_chars * BYTES_ONE_CHAR`

galexandrus replied:
Изменения внесены.

EgorOb reviewed (approved):
Зачтено

galexandrus committed:
Update task4

"total_bytes = total_chars * 1" 
replace with 
"total_bytes = total_chars * BYTES_ONE_CHAR"
